jsonrpc/types.go

```diff
@@ -344,3 +344,37 @@ type progression struct {
 	CurrentBlock  argUint64 `json:"currentBlock"`
 	HighestBlock  argUint64 `json:"highestBlock"`
 }
+
+type feeHistoryResult struct {
```

Comment:
What's this type used for? I guess this should be used in place of `FeeHistoryReturn`